src/main/java/frc/robot/Constants.java

```diff
@@ -1,38 +1,35 @@
-package frc.robot;
+// package frc.robot;
 /*
  * Constant values used throughout robot code.
  * In "C" this would be a header file, but alas, this is Java
  */
-public class Constants {
+// @Deprecated
+// public class Constants {
```

Comment:
Let's reinstate the Constants class and create enumerations for each of the general constants categories. 
Use the enum values to index a dictionary that holds the string key for the constant.
You can then encapsulate the config store inside the Constants class, thereby abstracting the whole mechanism for retrieving constants.

Reply:
I made the enum thing work.